content/en/profiler/profiler_troubleshooting/dotnet.md

```diff
@@ -138,17 +138,20 @@ Otherwise, turn on [debug mode][1] and [open a support ticket][2] with the debug
 
 ## Reduce overhead when using the profiler
 
-### Enabling the profiler machine-wide
+The different profilers have a fixed cpu and memory overhead. Because such a fixed overhead per profiled application exists, *the more profiled applications, the higher the overhead!*
 
-Datadog does not recommend enabling the profiler at machine-level or for all IIS application pools, as the profiler has fixed overhead per profiled application. In order to reduce the amount of resources used by the profiler, you can:
+### Avoid enabling the profiler machine-wide
+
+Datadog does not recommend enabling the profiler at machine-level or for all IIS application pools. In order to reduce the amount of resources used by the profiler, you can:
 - Increase the allocated resources, such as increasing CPU cores.
 - Profile only specific applications by setting environment in batch files instead of directly running the application.
 - Reduce the number of IIS pools being profiled (only possible in IIS 10 or later).
 - Disable wall time profiling with the setting `DD_PROFILING_WALLTIME_ENABLED=0`.
 
 ### Linux Containers
```

Comment:
More as a question to @gturbat, should we move this to be more generic to profiling? E.g. I guess a few of our profilers can have a bigger impact on really really tiny containers, not just .net?

Reply:
I assume you meant to comment the header  `### Linux Containers` below?